app/helpers/hosts_helper.rb

```diff
@@ -517,4 +517,19 @@ def power_status_visible?
   def host_breadcrumb
     breadcrumbs(resource_url: "/api/v2/hosts?thin=true'")
   end
+
+  def remediation_help_text
+    case @host.build_status_label
```

Comment:
Please use `build_status` method instead and compare constants rather than text.